plugins/linux-capture/pipewire.c

```diff
@@ -655,6 +708,12 @@ static void play_pipewire_stream(obs_pipewire_data *obs_pw)
 	pw_core_add_listener(obs_pw->core, &obs_pw->core_listener, &core_events,
 			     obs_pw);
 
+	// Dispatch to receive the info core event
+	obs_pw->server_version_sync = pw_core_sync(obs_pw->core, PW_ID_CORE,
+						   obs_pw->server_version_sync);
+	pw_thread_loop_wait(obs_pw->thread_loop);
+	obs_pw->server_version_sync = 0;
```

Comment:
Looking at various examples in the PipeWire codebase, Wim doesn't do this with any sync variables - I think we can just ignore `server_version_sync` after it's used

Reply:
This was just done to keep compatibility with the obsolete check from the review above.